src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -99,8 +99,11 @@ object IndexConstants {
   // Default id used for a file which does not have an id or its id is not known.
   private[hyperspace] val UNKNOWN_FILE_ID: Long = -1L
 
-  // JSON property name used in index metadata to store whether lineage is enabled for an index.
+  // JSON property names used in index metadata.
+  // Indicate whether lineage is enabled for an index.
   private[hyperspace] val LINEAGE_PROPERTY = "lineage"
+  // Indicate whether the source file format is parquet.
+  private[hyperspace] val IS_PARQUET_SOURCE_PROPERTY = "isParquetSource"
```

Comment:
Can we just name this as `val HAS_PARQUET_AS_SOURCE_FORMAT = "hasParquetAsSourceFormat"`?

Reply:
Oops, sorry, I meant to keep the `_PROPERTY` at the end to be consistent with `LINEAGE_PROPERTY`: `HAS_PARQUET_AS_SOURCE_FORMAT_PROPERTY`.